### PR TITLE
Make top/left of window dispatch events on updates

### PR DESCRIPTION
**Pull request by tshirtman**
tentative fix for #6132 , but doesn't seem to work

### CODE REVIEW

**Comment by matham** [comment]
What's the status of this?

**Comment by akshayaurora** [comment]
@matham  did some changes tested with window_sdl2 on osx and It works as expected.

File: kivy/properties.pyx [hunk 1]
```diff
@@ -1466,11 +1466,8 @@ cdef class AliasProperty(Property):
     cpdef link_deps(self, EventDispatcher obj, str name):
         cdef Property oprop
         for prop in self.bind_objects:
-            if prop in obj.__events__:
-                obj.fbind(prop, self.trigger_change, 0)
-            else:
-                oprop = getattr(obj.__class__, prop)
-                oprop.fbind(obj, self.trigger_change, 0)
```
Comment (akshayaurora) [inline kivy/properties.pyx]: sry, for removing this, this was indeed needed.

File: kivy/core/window/__init__.py [hunk 2]
```diff
@@ -1431,6 +1433,9 @@ def on_resize(self, width, height):
         '''Event called when the window is resized.'''
         self.update_viewport()
 
+    def on_move(self):
+        self._moved = not self._moved
+
```
Comment (matham) [inline kivy/core/window/__init__.py]: Why do you need the `_moved` property? Could you not just instead directly dispatch `top`  and `left` here (`self.dispatch('top')`).

**Pull request merged**